frontend/src/pages/Status.tsx

```diff
@@ -55,6 +55,32 @@ export function hasFinished(status?: NodePhase): boolean {
   }
 }
 
+export function statusToFadedColor(status?: NodePhase): string {
+  if (!status) {
```

Comment:
Is this needed? Shouldn't this be handled in the `default` case below?

Reply:
Correct; removed